public/views/address.html

```diff
@@ -1,9 +1,9 @@
 <section data-ng-controller="AddressController">
+  <link rel=StyleSheet href="../src/css/common.css" type="text/css" media="screen">
   <div class="secondary_navbar hidden-xs hidden-sm" scroll data-ng-class="{'hidden': !secondaryNavbar}" data-ng-if="address.balance >= 0" data-ng-init="hideSNavbar=0">
     <div class="container" data-ng-if="!hideSNavbar">
       <div class="col-md-8 text-left">
-        <h3>Address</h3> {{address.address}}
-        <span class="btn-copy" clip-copy="address.address"></span>
+        <h2>Address:&nbsp;{{address.address}}</h2>
```

Comment:
I chose this way because it matches with the themes of the transaction page among others